pyproject.toml

```
@@ -32,9 +32,12 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.10,<3.13"
+python = ">=3.8,<3.13"
 antlr4-python3-runtime = "^4.13.1"
-numpy = "^1.26.1"
+numpy = [
+    { version = "1.24", python = "<3.12" },
+    { version = "^1.26", python = "3.12" },
```

Given that 1.26 works for everything 3.9 and later, wouldn't this be better?

```suggestion
    { version = "1.24", python = "<3.9" },
    { version = "^1.26", python >= "3.9" },
```

---

you are absolutely right it's fixed